### ISSUE TITLE
Robot programming Day #32

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#59]
**Today, May 6, 2024, @DanielRomanRodriguez I am at home and I will be programming the robot at my house, I took the robot and the track to program.**

- The goal today is to make the robot make at least one perfect turn.

**Comment by WROMagnet** [comment]
Before starting to program I will arrange all the cables so that everything looks more organized here with a photo.



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/9ad569ef-2b7e-4761-859b-b0b3cc15e79e)



![Image](https://github.com/WROMagnet/Magnet-Robots/assets/166937907/64e0c7fe-91b7-4c1c-ac2a-d929964b7dc1)

**Comment by WROMagnet** [comment]
Now I'll start programming so that when I detect the orange color I'll take a turn and see how the programming goes.

- I have just started programming and something I have noticed is that the robot is going too fast which causes the sensor to not detect the colors in time. So I thought about seeing if I could slow down the robot to see if it could detect it.

- I already did the test by lowering the speed of the DC Motors and put this command:

`prizm.setMotorPowers(-8,8);`

But it didn't work, even though the robot is too fast for the sensor to detect the colors in time to do the action.

I plan to lower the sensor to see if by chance it is too far from the ground but I don't have the materials to lower the sensor but I do have materials to move the sensor forward a little so I will move the sensor forward a little.

- Ya monte el sensor un poco mas afuera asi que mañana estaremos montando el sensor de color un poco mas abajo aver si logra detectar el color a tiempo.

_With this I end this day since I do not have materials to lower the sensor so tomorrow we will be lowering the color sensor._
